scene/2d/collision_object_2d.cpp

```diff
@@ -28,7 +28,8 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                 */
 /**************************************************************************/
 
-#include "collision_object_2d.h"
+#include "scene/2d/collision_object_2d.h"
+#include "scene/2d/path_2d.h"
 
```

Comment:
```suggestion
#include "collision_object_2d.h"

#include "scene/2d/path_2d.h"
```
Maybe I was unclear what I meant